sqs-dead-letter-requeue/main.go

```diff
@@ -11,15 +11,20 @@ import (
 
 var (
 	app       = kingpin.New("dead-letter-requeue", "Requeues messages from a SQS dead-letter queue to the active one.")
-	queueName = app.Arg("queue-name", "Name of the SQS queue (e.g. prod-mgmt-website-data-www100-jimdo-com).").Required().String()
+	queueName = app.Arg("queue-name", "Name of the destination SQS queue (e.g. prod-mgmt-website-data-www100-jimdo-com).").Required().String()
+	fromQueueName = app.Arg("from-queue-name", "Name of the source SQS queue (e.g. prod-mgmt-website-data-www100-jimdo-com-dead-letter).").String()
 )
 
 func main() {
 	kingpin.MustParse(app.Parse(os.Args[1:]))
 
-	activeQueueName := *queueName
-
-	var deadLetterQueueName = activeQueueName + "_dead_letter"
+	destinationQueueName := *queueName
+		
```

Comment:
...nope, you don't due to this `:=` thing... OMG my go skills have died a slow and painfull death. 😢